### ISSUE TITLE
How to set bodyParser limit?

### ISSUE THREAD

**Issue by jenssogaard** [zeit/next.js#4233]
- [x] I have searched the [issues](https://github.com/zeit/next.js/issues?q=is%3Aissue) of this repository and believe that this is not a duplicate.

I need to bump up the default 100KB [bodyParser raw.js limit](https://github.com/expressjs/body-parser/blob/master/lib/types/raw.js) but I can't seem to figure out the right way in Next.js 5.1.0
I tried adding it to index.js but with no effect:
`expressApp.use(bodyParser.raw({ limit: '5mb' }));`

I also modified the [raw.js](https://github.com/expressjs/body-parser/blob/master/lib/types/raw.js) vendor file to make sure it would work if the value was increased.

Long story short, how would I go about modifying the raw body-parser limit?
`   
PayloadTooLargeError: request entity too large
    at readStream (/Users/xxx/node_modules/raw-body/index.js:155:17)
    at getRawBody (/Users/xxx/node_modules/raw-body/index.js:108:12)
    at read (/Users/xxx/node_modules/body-parser/lib/read.js:77:3)
    at jsonParser (/Users/xxx/node_modules/body-parser/lib/types/json.js:134:5)
    at Layer.handle [as handle_request] (/Users/xxx/node_modules/express/lib/router/layer.js:95:5)
`

**Comment by timneutkens** [comment]
Next.js doesn't use express by itself, and anything you do with express doesn't touch Next.js until you call app.render

It's also better to implement APIs separately from the Next.js server, both for seperation of concerns and performance.

**Comment by jenssogaard** [comment]
Thanks @timneutkens. Makes sense.

**Comment by ghengeveld** [comment]
Another tip: don't enable bodyparser globally, but only for the specific routes that need it. Improves performance and avoids some headaches with incompatible middleware.
